build.sbt

```diff
@@ -27,10 +28,24 @@ lazy val builderArgument = settingKey[String](
 lazy val releaseTask = taskKey[Unit](
   "Publishes an artifact and optionally makes a release if version is not a snapshot")
 
+def latestRelease: String = {
+  val src = scala.io.Source.fromURL("https://api.github.com/repos/evansiroky/timezone-boundary-builder/releases/latest")
+  val json = src.mkString
+  json
+}
+
+def parseReleaseName(jsonString: String): String = {
+  val json:Option[Any] = JSON.parseFull(jsonString)
+  val map:Map[String,Any] = json.get.asInstanceOf[Map[String, Any]]
+  val name:String = map("name").asInstanceOf[String]
+  println("Current timezone release is : " + name)
+  name
+}
+
```

Comment:
Looks way better!

Reply:
I was playing with the code because we need a update with new timezone data because in 2019 Brazil will not have daylight savings and soon our time display would break.

Because of this I was looking for a stopgap solution until a new maven release.